### PR TITLE
Send welcome mail called from frm

### PR DESCRIPTION
**Pull request by 0925-pinka**
Peek 2020-02-11 17-38

### CODE REVIEW

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/mail_67/frappe/pullRequest?prid=5014039&bid=16535177)

File: frappe/core/doctype/user/user.js [hunk 1]
```diff
@@ -73,15 +73,12 @@ frappe.ui.form.on('User', {
...
-						},
-					})
-				})
-
+				frm.add_custom_button(__("Resend Welcome Email"), function() {
+					frm.call('send_welcome_mail_to_user').then(()=>{
+						frappe.msgprint(__("Email has been sent to {0}", [frm.doc.email]));
+					});
+				});
+
```
Comment (vishdha) [inline frappe/core/doctype/user/user.js:81]: Remove tab space

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/mail_67/frappe/pullRequest?prid=5014039&bid=16535177)

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/mail_67/frappe/pullRequest?prid=5014039&bid=16535177)

**Review by Alchez — APPROVED** [review]

**Pull request merged**